install_config/upgrading/blue_green_deployments.adoc

```diff
@@ -175,6 +175,17 @@ xref:../../architecture/infrastructure_components/kubernetes_infrastructure.adoc
 (which you will verify in later steps), set the `openshift_schedulable=false`
 variable for each node host to ensure they are unschedulable initially.
 
+.Example of Adding New Nodes
```

Comment:
Maybe the title here should be `Example *new_nodes* Host Group` ?

We should make a note here that the example below should be added to their existing inventory, everything that was in there previously should remain.